controllers/cloud.redhat.com/providers/utils/utils.go

```diff
@@ -15,6 +15,20 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+func DatabaseResourceDefaults() core.ResourceRequirements {
```

Comment:
Don't need to address this now ... but IMO we should have a follow-up PR that obtains this value from a configuration block in the ClowdEnvironment... so that we can adjust the defaults if desired without needing a new code push.